app/views/admin/fleets/index.html.haml

```diff
@@ -0,0 +1,14 @@
+%h3 Fleets
+= link_to 'New Fleet', new_admin_fleet_path
+
+%table
+  %tr
+    %th Fleet Name
+    %th Trucks
+  - @fleet.each do |fleet|
+    %tr
+      %td= fleet.fleet_name
+      %td= fleet.trucks.map{|truck| truck.registration }.join(", ")
```

Comment:
This can be a little more concise as `fleet.trucks.map(&:registration).join(", ")`